src/python/ensembl/io/genomio/seq_region/dump.py

```diff
@@ -39,8 +40,9 @@
 from ensembl.utils.logging import init_logging_with_args
 
 
-_ROOT_DIR = Path(__file__).parent / "../../../../../.."
-_DEFAULT_MAP = _ROOT_DIR / "config/external_db_map/default.txt"
+default_map_res = files("ensembl.io.genomio.data.external_db_map").joinpath("default.txt")
+with as_file(default_map_res) as default_map_path:
+    _DEFAULT_MAP = default_map_path
```

Comment:
This is quite handy, isn't it? Although I'm a little bit annoyed that you have to use the `with` now to access the file, when you didn't before.

Reply:
RIght, I think I need to actually test this because I would expect the resource/file to become unavailable after the with context is closed...

Reply:
I guess that's the price to pay to have access to resources that could be something else than files.